.gitpod.yml

```diff
@@ -0,0 +1,28 @@
+tasks:
+  - name: Serve Sphinx Docs
+    init: |
+      cd /workspace/aac-gherkin/docs/
+      sleep 120 # Gitpod often takes a long time to install the python environment
+      source /workspace/AaC/python/venv/bin/activate
+      sphinx-build source gen_docs # Pre-build the docs so that when Gitpod opens the port it's ready to serve
+    command: sphinx-autobuild source gen_docs
+
+  - name: Python Virtual Environment
+    init: |
+      cd python/
+      python3.9 -m venv venv --system-site-packages
+      source venv/bin/activate
+      pip install -e .[all]
+    command: cd /workspace/aac-gherkin/python/; source venv/bin/activate # Always launch a console with our virtual env

```

Comment:
This should be referencing the repo as part of the path, so should be `aac-gen-gherkin`